src/router.js

```diff
@@ -6,20 +6,23 @@ import Home from './views/Home.vue'
 
 Vue.use(Router)
 
-export default new Router({
-  routes: [
-    {
-      path: '/signup',
-      component: Signin
-    },
-    {
-      path: '/login',
-      component: Login
-    },
-    {
-      path: '/home',
-      component: Home
-    }
-  ],
-  mode: 'history'
+const privateRoutes = [{ path: '/signup', component: Signin }, { path: '/login', component: Login }]
+const publicRoutes = [{ path: '/home', component: Home }]
```

Comment:
This is a nice approach, but you can also add extra info to the routes, such as if it's private or not with the meta field https://router.vuejs.org/guide/advanced/meta.html

```js
{ path: '/home', component: Home, meta: { isPrivate: false } }
```